website/src/controllers/api-v3/groups.js

```diff
@@ -563,15 +520,25 @@ api.inviteToGroup = {
     let uuids = req.body.uuids;
     let emails = req.body.emails;
 
-    if (uuids && emails) { // TODO fix this, low priority, allow for inviting by both at the same time
-      throw new BadRequest(res.t('canOnlyInviteEmailUuid'));
-    } else if (Array.isArray(uuids)) {
-      // return _inviteByUUIDs(uuids, group, user, req, res, next);
-    } else if (Array.isArray(emails)) {
-      // return _inviteByEmails(emails, group, user, req, res, next);
-    } else {
+    if (!Array.isArray(uuids) && !Array.isArray(emails)) {
```

Comment:
Yeah it looks good, I misread the code. One thing that may change is the error string here https://github.com/HabitRPG/habitrpg/pull/6475/files#diff-110878cdf614e13dc8eac3ea77ed6265R524 

And when you push to results you may want to use the spread operator to push invites individually to `results`: `results.push(...uuidResults)` More infot https://developer.mozilla.org/it/docs/Web/JavaScript/Reference/Operators/Spread_operator